### ISSUE TITLE
➤ Alina Moldovan commented:

### ISSUE THREAD

**Issue by thotha** [mozilla-mobile/firefox-ios#16157]
➤ Alina Moldovan commented:

I tried to validate this bug using FF v116.3 (33647) and iPhone X (16.6), iPhone 14+ (16.0.3) and iPad 3rd Gen (17.0)
I was not able to reproduce this bug.

_Originally posted by @data-sync-user in https://github.com/mozilla-mobile/firefox-ios/issues/15885#issuecomment-1686316663_

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-7277)

**Comment by thotha** [comment]
OK, then maybe some one can guide me?
I was able to extract the folder "AppDomainGroup-group.org.mozilla.ios.Firefox" from within "Firefox -> Shared" from backups back to May 29th 2023. 
How do I extract the plain HTML data from the database file "profile.profile/tabs.db" and/or "profile.profile/browser.db" or any other DB where all open tab's are stored. That way I'm maybe able to find the URL's in question.
I have to say I don't know SQL but if you write how to do it I'm able to execute the commands, even if it's teh CLI.

With 60y, my brain get's more and more holes like a "Swiss Emmentaler Cheese", but I used to work as supporter for Mac Computers at a University and therefore I do know this ore the other thing.
